common/src/main/aidl/com/microsoft/identity/client/IMicrosoftAuthService.aidl

```diff
@@ -32,8 +32,12 @@ interface IMicrosoftAuthService {
 
     Bundle hello(in Bundle bundle);
 
+    Bundle getAccountMode();
```

Comment:
Question, who is setting the account mode? Broker or MSAL application?

Reply:
Broker. 

Here's roughly what happens in FLW scenario
1. Admin bootstrap a device with their 'admin' WPJ account through Authenticator app.
2. WPJ flows sends the flag to the server side, the server validates WPJ account's role.
3. If the role matches, the WPJ operation succeeds and the device is marked as shared - therefore a 'mode' is set.
4. MSAL queries this 'mode' to determine which set of Broker functionality it should expose to MSAL application.